### DISCUSSION TITLE
为什么阿里的lowcode-engine项目不是基于designable的？

### DISCUSSION THREAD

**Discussion by xiemeilong** [alibaba/designable#245] [General]
发现阿里刚开源不久的[lowcode-engine](https://github.com/alibaba/lowcode-engine)项目并不是基于designable的，但整个界面又很类似。是什么情况，两个项目将来会一直独立发展？

**Comment by janryWang** [comment]
dn是设计器引擎，不负责包装产品，所以dn的定位更像一个library，它后面的路线是尽可能的优化设计器的各种设计能力，至于上层产品如何封装，是由用户自己发挥的

**Reply by sstruct** [reply]
> dn是设计器引擎，不负责包装产品，所以dn的定位更像一个library，它后面的路线是尽可能的优化设计器的各种设计能力，至于上层产品如何封装，是由用户自己发挥的

同问，lowcode-engine 代码和文档里似乎没有提到 formily/designable，但看起来却很相似，两者设计或代码上有关联吗

**Reply by janryWang** [reply]
没啥关联

**Comment by xiemeilong** [comment]
对比了下lowcode-engine和designable，我更喜欢designable（也更喜欢antd和formily以及函数式组件），但是lowcode-engine又做的更全面，纠结了。
